lib/jellyfish/application.ex

```diff
@@ -19,7 +20,8 @@ defmodule Jellyfish.Application do
       JellyfishWeb.Endpoint,
       # Start the RoomService
       Jellyfish.RoomService,
-      {Registry, keys: :unique, name: Jellyfish.RoomRegistry}
+      {Registry, keys: :unique, name: Jellyfish.RoomRegistry},
+      {Cluster.Supervisor, [topologies, [name: Jellyfish.ClusterSupervisor]]}
```

Comment:
Just curious -- does this mean that we'll always want to start the `Cluster.Supervisor`, even when not actually using distribution?

Reply:
If I understand correctly how Supervisor in Beam works, then yes.